docs/sdk/getting-started/react-native.md

```diff
@@ -133,10 +133,10 @@ Do this integration, if you link React modules using relative path inside your P
 1. Inside your Podfile, insert the following lines:
 
     ```ruby
-    pod 'appcenter', path: '../node_modules/appcenter'
-    pod 'appcenter-analytics', path: '../node_modules/appcenter-analytics'
-    pod 'appcenter-crashes', path: '../node_modules/appcenter-crashes'
-    pod 'appcenter-push', path: '../node_modules/appcenter-push'
+    pod 'appcenter', path: '../node_modules/appcenter/ios'
```

Comment:
Hi @Kerizer,

This section of the doc explains how to integrate App Center SDK using relative path. In order for cocoapods to know the local relative path of App Center SDK, developers will have to specify the SDK's **podspec** file path, for example: `pod 'appcenter-analytics', path: '../node_modules/appcenter-analytics'` where as **podspec** located at **../node_modules/appcenter-analytics/ios/appcenter-analytics.podspec**

It doesn't really matter whether the path is `'../node_modules/appcenter-analytics'` OR `'../node_modules/appcenter-analytics/ios'` because they both work. The former one allows developers to use customized path for their React Native iOS project other than {ProjectRoot}/ios.

Why do you proposed to change the path?

Reply:
Thsi is what you get if path is `../node_modules/appcenter` instead of `../node_modules/appcenter-analytics/ios` 
```
[!] No podspec found for `appcenter` in `../node_modules/appcenter`
```
 https://ibb.co/znjZbzBth

Reply:
Hi @Kerizer,

Sorry for the confusion. A colleague of mine verified this behavior and told me that you are right about this. My last reply comes from my previous experience with this and it's probably not up to date.

Reply:
Hi @Kerizer,

Thanks for the contribution, we really appreciate it.